hazelcast/src/main/java/com/hazelcast/quorum/QuorumChangeStateOperation.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.quorum;
+
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.spi.Operation;
+import java.io.IOException;
+
+/**
+ * Changes quorum state to the given state on the executing node.
+ */
+public class QuorumChangeStateOperation extends Operation {
```

Comment:
I would suggest extending from AbstractOperation to reduce the amount of empty methods that need to be implemented.